### PR TITLE
UAC-free Sourcetree installer

### PR DESCRIPTION
**Pull request by damnhandy**
I've run into a few issues with SourceTree needing admin rights in order to be installed. I took a gander at this repository:

   https://github.com/cosmomill/SourceTreePortable

And attempted to refactor it into a PowerShell script. My PowerShell is weak, but the user is not promoted with a UAC prompt. Is there a better way to this>

### CODE REVIEW

**Comment by lukesampson** [comment]
Sounds good to me. @vidarkongsli can you see any problems with this approach? If not, I'll merge it in.

File: scripts/sourcetree.ps1 [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# Unpack the installer to expose the MSI
+$unpack = Start-Process "$dir\$sourcetreeIns.exe" -Wait -ea Stop -PassThru -arg "/extract"
+if($unpack.ExitCode -eq 0)
+{
+    # The extracted MSI doesn't seem to like being installed in $dir, so we put it in subditectory
+    mkdir $dir\app
+    $install = Start-Process msiexec -Wait -ea stop -PassThru -arg "/a $dir\$sourcetreeIns.msi /qb TARGETDIR=$dir\app"
+    if($install.ExitCode -ne 0)
+    {
+        Write-Host "Installer failed."
```
Comment (vidarkongsli) [inline scripts/sourcetree.ps1]: I suggest using `Write-Error` here.

**Comment by vidarkongsli** [comment]
:+1:

**Comment by damnhandy** [comment]
Done.

**Comment by lukesampson** [comment]
Thanks :smile:

**Pull request merged**